### PR TITLE
Replace mvn deploy command with gcloud deploy in deploy.sh

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR replaces the old mvn deploy command with the manual deployment steps.

- <code>mvn clean package</code> builds the application
- <code>gcloud app deploy</code> deploys the application on gcloud.

### CODE REVIEW

**Review by MagoDelBlocco — APPROVED** [review]
LGTM

File: deploy.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 gcloud config set project llvm-build-dashboard &&
-mvn package com.google.cloud.tools:appengine-maven-plugin:deploy
+mvn clean package &&
```
Comment (krasimir-google) [inline deploy.sh:3]: a suggestion for a slight improvement: you can start the bash script with [`set -e`](https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script) and remove all the trailing `&&`-s. This set option has the effect of exiting as soon as the first command executed returns a non-zero exit status.

**Review by krasimir-google — APPROVED** [review]
LGTM I've got a suggestion for this bash script that can be done in a separate PR

File: deploy.sh (hunk 1 above)
Comment (samuelsogbesan) [reply deploy.sh:3]: Awesome!

**Pull request merged**